maestro-cli/src/main/java/maestro/cli/session/MaestroSessionManager.kt

```diff
@@ -0,0 +1,265 @@
+/*
+ *
+ *  Copyright (c) 2022 mobile.dev inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *
+ */
+
+package maestro.cli.session
+
+import dadb.Dadb
+import dadb.adbserver.AdbServer
+import io.grpc.ManagedChannelBuilder
+import ios.idb.IdbIOSDevice
+import maestro.Maestro
+import maestro.cli.device.Device
+import maestro.cli.device.PickDeviceInteractor
+import maestro.cli.device.Platform
+import maestro.drivers.IOSDriver
+import java.util.UUID
+import java.util.concurrent.Executors
+import java.util.concurrent.TimeUnit
+import kotlin.concurrent.thread
+
+object MaestroSessionManager {
+    private const val defaultHost = "localhost"
+    private const val idbPort = 10882
```

Comment:
I believe this won't work for multiple iOS sessions yet because there needs to be multiple IDB instances with different port numbers available.

Reply:
The idea is that it should work for a single iOS device, in that case we can reuse the same idb instance